records_mover/records/prep_and_load.py

```diff
@@ -16,24 +16,24 @@ def prep_and_load(tbl: TargetTableDetails,
                   load_exception_type: Type[Exception],
                   reset_before_reload: Callable[[], None] = lambda: None) -> MoveResult:
     logger.info("Connecting to database...")
-    with tbl.db_engine.begin() as db:
-        driver = tbl.db_driver(db)
+    with tbl.db_engine.begin() as db_conn:
```

Comment:
can you do this? db_engine.begin() does it create a connection and start a transaction all at once?

Reply:
i think it would be better to get the connection from the engine, then use `conn.begin()` the same way the majority if the rest of the code does  it - our overarching goal is treating engines like engines and connections like connections and not mixing them even if sqlalchemy allows us to.

Reply:
This makes sense, easy rewrite! I was just trying to retain as much of the original writing as possible.